src/sentry/api/event_search.py

```diff
@@ -645,20 +646,21 @@ def get_snuba_query_args(query=None, params=None):
             snuba_name = term.key.snuba_name
             if term.key.name == PROJECT_KEY:
                 condition = ['project_id', '=', projects.get(term.value.value)]
-                kwargs['conditions'].append(condition)
+                conditions.append(condition)
 
-            elif snuba_name in ('start', 'end'):
-                kwargs[snuba_name] = term.value.value
+            elif snuba_name == 'start':
+                start = term.value.value
+            elif snuba_name == 'end':
+                end = term.value.value
             elif snuba_name in ('project_id', 'issue'):
                 value = term.value.value
                 if isinstance(value, int):
                     value = [value]
-                kwargs['filter_keys'][snuba_name].extend(value)
+                filter_keys[snuba_name].extend(value)
             else:
                 converted_filter = convert_search_filter_to_snuba_query(term)
-                kwargs['conditions'].append(converted_filter)
-        else:  # SearchBoolean
-            # TODO(lb): remove when boolean terms fully functional
-            kwargs['has_boolean_terms'] = True
-            kwargs['conditions'].append(convert_search_boolean_to_snuba_query(term))
-    return kwargs
+                conditions.append(converted_filter)
+        # else:  # SearchBoolean
+            # conditions.append(convert_search_boolean_to_snuba_query(term))
```

Comment:
Any reason not to turn this on?

Reply:
I'm not sure if boolean search is ready to go, and I don't want to turn it on in this PR